_i18n/es/resources/user-guides/monerod-as-a-systemd-service.md

```diff
@@ -0,0 +1,73 @@
+{% include untranslated.html %}
+# Using monerod as a systemd Service
+
+If you are using a Linux distribution that uses `systemd`, you can use it's
+user units to manage `monerod` as well.
+
+We will do this in the command line & assume that you have already built or
+downloaded `monerod`.
+
+
+## Create the Service File
+
+First start by creating the directory for your user service:
+
+    mkdir -p ~/.config/systemd/user/
+
+Then create `~/.config/systemd/user/monerod.service` with the following content:
+
+    [Unit]
+    Description=Monero Daemon
+
+    [Service]
+    ExecStart=/your/path/to/monerod --non-interactive
```

Comment:
Not sure of the advantages but the example unit definition file included with the Monero code (`utils/systemd/monerod.service`) uses `Type=forking` and the `--detach` parameter.

Reply:
Didn't notice there was an official systemd service already since I grabbed the getmonero.org binaries.

Seems like you actually get `journalctl` & `systemctl status` output with `non-interactive`:

```
Feb 13 12:36:04 crypto systemd[1336]: Started Monero Blockchain Daemon.
Feb 13 12:36:04 crypto monerod[23988]: 2018-02-13 17:36:04.017            7f36ab5fa740        INFO         global        src/daemon/main.cpp:279        Monero 'Helium Hydra' (v0.11.1.0-release)
Feb 13 12:36:04 crypto monerod[23988]: 2018-02-13 17:36:04.017            7f36ab5fa740        INFO         global        src/daemon/protocol.h:55        Initializing cryptonote protocol...
Feb 13 12:36:04 crypto monerod[23988]: 2018-02-13 17:36:04.017            7f36ab5fa740        INFO         global        src/daemon/protocol.h:60        Cryptonote protocol initialized OK
Feb 13 12:36:04 crypto monerod[23988]: 2018-02-13 17:36:04.018            7f36ab5fa740        INFO         global        src/daemon/p2p.h:63        Initializing p2p server...
Feb 13 12:36:08 crypto monerod[23988]: 2018-02-13 17:36:08.293            7f36ab5fa740        INFO         global        src/daemon/p2p.h:68        P2p server initialized OK
...
```

Vs. `Type=forking`:
```
Feb 13 12:33:08 crypto systemd[1336]: Starting Monero Blockchain Daemon...
Feb 13 12:33:08 crypto monerod[23089]: 2018-02-13 17:33:08.647            7fe3665f0740        INFO         global        src/daemon/main.cpp:279   
Feb 13 12:33:08 crypto monerod[23089]: Forking to background...
Feb 13 12:33:08 crypto systemd[1336]: monero.service: Supervising process 23091 which is not our child. We'll most likely not notice when it exits.
Feb 13 12:33:08 crypto systemd[1336]: Started Monero Blockchain Daemon.
```

But I could change it to be consistent with the service file in the `monero` repo.

Reply:
In the config they use with the service definition in the repo, they set it to log at /var/log/monero/monero.log I think. Not sure what you should do, maybe somebody else knows better :)